reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -14,66 +14,168 @@ Explains language modes and their effect on PowerShell sessions.
 
 ## Long description
 
-The language mode of a PowerShell session determines, in part, which elements
-of the PowerShell language can be used in the session.
+The language mode of a PowerShell session determines which elements of the
+PowerShell language can be used in the session.
 
 PowerShell supports the following language modes:
 
-- **FullLanguage**
-- **ConstrainedLanguage** (introduced in PowerShell 3.0)
-- **RestrictedLanguage**
-- **NoLanguage**
+- `FullLanguage`
+- `RestrictedLanguage`
+- `ConstrainedLanguage` (introduced in PowerShell 3.0)
+- `NoLanguage`
 
-### What is a language mode?
+## What is a language mode?
 
 The language mode determines the language elements that are permitted in the
 session.
 
-The language mode is actually a property of the session configuration (or
-"endpoint") that is used to create the session. All sessions that use a
-particular session configuration have the language mode of the session
+The language mode is a property of the session configuration (or "endpoint")
+that's used to create the session. All sessions that use a particular session
+configuration have the language mode of the session configuration.
+
+All PowerShell sessions have a language mode. Remote sessions are created using
+the session configurations on the remote computer. The language mode set in the
+session configuration determines the language mode of the session. To specify
+the session configuration of a PSSession, use the **ConfigurationName**
+parameter of cmdlets that create a session.
```

Comment:
Suggest using target instead of remote here since they can be used for local sessions too.
```suggestion
All PowerShell sessions have a language mode. Sessions are created using the
session configurations on the target computer. The language mode set in the
session configuration determines the language mode of the session. To specify
the session configuration of a PSSession, use the **ConfigurationName**
parameter of cmdlets that create a session.
```